site/content/contribute/server/plugins.md

```diff
@@ -20,7 +20,7 @@ Plugins are generally made of at least two parts: a manifest and a server binary
 
 The manifest tells Mattermost what the plugin is and provides a set of metadata used by the server to install and run the plugin. Please see the [manifest reference]({{< ref "/integrate/plugins/manifest-reference" >}}) for more information. Manifests may be defined in JSON or YAML.
 
-The server binary is a compiled (generally Go) program that extends the [MattermostPlugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#MattermostPlugin) struct of the [plugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin) package. When enabled, the plugin's server binary is started as a process by the Mattermost server. Plugin builders then have access to interact with the Mattermost server over RPC through the plugin [API](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#API) and [Hooks](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#Hooks). The server-side of plugins is built using the [go-plugin](https://github.com/hashicorp/go-plugin) library from Hashicorp. More information is available in the [server side of the plugin author documentation]({{< ref "/integrate/plugins/server" >}}).
+The server binary is a compiled go program that extends the [MattermostPlugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#MattermostPlugin) struct of the [plugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin) package. When enabled, the plugin's server binary is started as a process by the Mattermost server. Plugin builders then have access to interact with the Mattermost server over RPC through the plugin [API](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#API) and [Hooks](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#Hooks). The server-side of plugins is built using the [go-plugin](https://github.com/hashicorp/go-plugin) library from Hashicorp. More information is available in the [server side of the plugin author documentation]({{< ref "/integrate/plugins/server" >}}).
```

Comment:
Should the language's name really be lowercased? JavaScript is capitalized below

Reply:
```suggestion
The server binary is a compiled go program that extends the [MattermostPlugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#MattermostPlugin) struct of the [plugin](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin) package. When enabled, the plugin's server binary is started as a process by the Mattermost server. Plugin creators (or authors?) then have access to interact with the Mattermost server over RPC through the plugin [API](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#API) and [Hooks](https://godoc.org/github.com/mattermost/mattermost-server/v6/plugin#Hooks). The server-side of plugins is built using the [go-plugin](https://github.com/hashicorp/go-plugin) library from Hashicorp. More information is available in the [server side of the plugin author documentation]({{< ref "/integrate/plugins/server" >}}).
```

https://developers.mattermost.com/integrate/plugins/server/reference seems more digestible as a reference for the RPC API. https://developers.mattermost.com/integrate/plugins/server/reference/#Hooks for hooks. But this is for contributing to the framework itself, so maybe not relevant.

Also, we never mention `mattermost-plugin-api` in this page nor the server reference page. There is a `Helpers` section there that is blank.